Tools/autotest/quadplane.py

```diff
@@ -166,8 +170,10 @@ def fly_qautotune(self):
         self.wait_disarmed()
 
     def takeoff(self, height, mode):
+        self.zero_throttle()
         self.change_mode(mode)
         self.wait_ready_to_arm()
+        self.zero_throttle()
```

Comment:
Is this really required?!

Reply:
RC3 tends to default to 1500, and that will fail the pre-arm check for "neutral"
As to whether all of these are required, I just inserted them everywhere after failing CI on half a dozen iterations.

Reply:
sorry, I hadn't noticed that this one was obviously redundant